corehq/apps/geospatial/utils.py

```diff
@@ -15,3 +22,53 @@ def get_geo_user_property(domain):
     except GeoConfig.DoesNotExist:
         config = GeoConfig()
     return config.user_location_property_name
+
+
+def _get_mapping(data_items):
+    return {
+        i['id']: f"{i['lat']} {i['lon']}"
+        for i in data_items
+        if i['lat'] and i['lon']
```

Comment:
could use better naming here to highlight what is being iterated like `case_data` instead of `i`.

Reply:
This function is no longer needed and has been removed entirely ([0c00723](https://github.com/dimagi/commcare-hq/pull/33420/commits/0c007232c7a461133dc8fd4ea0c971d273c659e0)).